QuizAppNew/View/QuestionView.swift

```diff
@@ -0,0 +1,141 @@
+                            .stroke(Color.black.opacity(0.5), lineWidth: 1)
+                    )
+            })
+            
+            //FOR OPTIONS....B
+            
+            Button(action: {}, label: {
+                Text("question.optionB")
+                    .foregroundColor(Color.black)
+                    .padding([.horizontal,.vertical],5)
+                    .frame(maxWidth: .infinity)
+                    .background(
+                        RoundedRectangle(cornerRadius: 15)
+                            .stroke(Color.black.opacity(0.5), lineWidth: 1)
+                    )
+            })
+            
+            //FOR OPTIONS....C
+            
+            Button(action: {}, label: {
+                Text("question.optionC")
+                    .foregroundColor(Color.black)
+                    .padding([.horizontal,.vertical],5)
+                    .frame(maxWidth: .infinity)
+                    .background(
+                        RoundedRectangle(cornerRadius: 15)
+                            .stroke(Color.black.opacity(0.5), lineWidth: 1)
+                    )
+            })
+            
+            //FOR OPTIONS....D
+            
+            Button(action: {}, label: {
+                Text("question.optionD")
+                    .foregroundColor(Color.black)
+                    .padding([.horizontal,.vertical],5)
+                    .frame(maxWidth: .infinity)
+                    .background(
+                        RoundedRectangle(cornerRadius: 15)
+                            .stroke(Color.black.opacity(0.5), lineWidth: 1)
+                    )
+            })
+            
+            Spacer(minLength: 0)
+            
+            HStack(spacing: 15) {
+                /* ---------
+                 Button(action: {
+                 self.showLoginView.toggle()
+                 }) {
+                 Text("Submit")
+                 .fontWeight(.heavy)
+                 .foregroundColor(.white)
+                 .padding(.vertical)
+                 .frame(maxWidth: .infinity)
+                 .background(Color.blue)
+                 .cornerRadius(15)
+                 }.sheet(isPresented: $showLoginView) {
+                 ResultView()
+                 }
+                 
+                 .fullScreenCover(isPresented: $showLoginView, content: {
+                 ResultView()
+                 })
+                 ----------- */
+                //
+                //                                Button("Submit") {
+                //                                    view.isSubmitted  = true
+                //                                      }
+                //                .sheet(isPresented: $showLoginView, content: {
+                //                          ResultView()
+                //                      })
+                //           //     Text("Submit")
+                Button(action: {
+                    showLoginView.toggle()
```

Comment:
**showLoginView** is there any login functionality? Please gives the meaningful variable name.